.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I do not think you wanted to upload this file :wink:

Reply:
🤦‍♂️